conf.py

```diff
@@ -0,0 +1,52 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
I think creating a makefile to build the sphinx documentation is within the scope of this task. We're adding a way to build the documentation so we should know _how_ to build it. (Creating a CI for it would be a follow-up however.) The simple one generated by `sphinx-quickstart` would do.